env/container/pg/Dockerfile

```diff
@@ -8,3 +8,6 @@ FROM base as dev
 
 ENV POSTGRES_USER 'dev'
 ENV POSTGRES_PASSWORD 'dev_secret'
+
+RUN useradd -ms /bin/bash user
```

Comment:
This one is take care of for us

https://github.com/docker-library/postgres/blob/master/10/Dockerfile#L18